modules/simple_bucket/variables.tf

```
@@ -54,6 +54,12 @@ variable "versioning" {
   default     = true
 }
 
+variable "autoclass" {
+  description = "While set to true, autoclass is enabled for this bucket."
+  type        = bool
+  default     = true
```

Unless autoclass is currently enabled by default, we should probably default this to `false` to maintain existing behavior (avoid breaking change).

```suggestion
  default     = false
```